lib/main.js

```diff
@@ -341,19 +449,45 @@ exports.main = function () {
     contentScript: 'self.on("click", self.postMessage)',
     onMessage: launchConfigTab
   });
+
   checkUpdate();
   setUpTopMenu(initSubMenus());
+
+  // Adding a simple playlist.
+  plists.playlist_id = plists.add_playlist();
+  plists.playlist_id = plists.add_playlist();
```

Comment:
I haven't checked the inner workings of this yet, but is there a specific reason it is done twice?